windows-installer/setup.nsi

```diff
@@ -16,7 +16,7 @@
 !define PRODUCT "Stackdriver"
 
 ; Software name used for display to users.
-!define DISPLAY_NAME "${COMPANY} ${PRODUCT} Logging Agent"
+!define DISPLAY_NAME "${COMPANY} ${PRODUCT} Logging Agent - ${VERSION}"
```

Comment:
I wouldn't bother adding the "-" separator. Just `"${COMPANY} ${PRODUCT} Logging Agent ${VERSION}"` should be fine.